schemas/2017-10-01/Microsoft.PowerBIDedicated.json

```diff
@@ -0,0 +1,94 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "definitions": {

```

Comment:
>definitions [](start = 3, length = 11)

we have a practice in the order of the these top level properties. can you please follow that?